package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node --experimental-json-modules -r dotenv/config ./populate-the-supabase/action.js"
```

Comment:
Maybe this script should be named something like "populate-db"?

Reply:
oh this shouldn't be here because of the `-r dotenv/config` part :D